vike-react/renderer/onRenderHtml.tsx

```diff
@@ -12,20 +12,16 @@ import React from 'react'
 checkVikeVersion()
 
 const onRenderHtml: OnRenderHtmlAsync = async (pageContext): ReturnType<OnRenderHtmlAsync> => {
-  let pageHtml = ''
-  if (!!pageContext.Page) {
```

Comment:
when `pageContext.Page` is undefined here this means that `ssr: false` is being used. Should we still be making use of `renderToStream()` in this case?